abodepy/event_controller.py

```diff
@@ -46,6 +50,36 @@ def stop(self):
         """Tell the subscription thread to terminate - will block."""
         self._socketio.stop()
 
+    def add_connection_status_callback(self, devices, callback):
```

Comment:
This method accepts not only `devices` but also `automations`. I'm not sure if it's worth changing this to `ids` to give it a more universal name, or just add a note.

Reply:
Renamed to `unique_id` which was required for fixing the issue with multi sensors. Resolves this issue.